src/menu/transientMenuItem.ts

```diff
@@ -44,7 +44,7 @@ export class TransientMenuItem implements TransientBindingItem, BaseWhichKeyMenu
     }
 
     get description(): string {
-        const icon = (this.showIcons && this.icon && this.icon.length > 0) ? `$(${this.icon}) ` : "";
+        const icon = (this.showIcons && this.icon && this.icon.length > 0) ? `$(${this.icon})   ` : "";
         return `\t${icon}${this._binding.name}`;
     }
-}
\ No newline at end of file
+}
```

Comment:
Wondering if you can remove these end of file changes?

Reply:
IMHO, it should be there (but probably in all files) because not having it breaks a lot of things, at least with typical Unix utilities - for example, `wc -l src/bufferCommands.ts` reports 10 lines (rather than 11) and when using `cat`, the shell prompt is stuck to the last line:

```
[florian@aragog src]$ cat constants.ts
[...]
export const defaultStatusBarTimeout = 5000;[florian@aragog src]$ 
```

Every line ends with a newline, so why not the last? :wink: Thus I have `"files.insertFinalNewline": true` set in VS Code.

Agreed this shouldn't be part of this PR though, so I'm going to revert them.

Reply:
I don't mind either way, happy to take PR ;) If we ever add newline to all files, we should also enforce it via `.eslintrc.js` with `'eol-last': 'error',` as rules.